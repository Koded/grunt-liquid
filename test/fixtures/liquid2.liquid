{% extends inc/layout %}

{% block body %}
  HELLO!
  
  {% include inc/head %}
{% endblock %}