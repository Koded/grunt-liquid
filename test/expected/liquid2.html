<html>
  <head>
  <title>A great title</title>
</head>
  <body>
    
  HELLO!
  
  <head>
  <title>A great title</title>
</head>

  </body>
</html>